packages/javascript/bh-shared-ui/src/utils/entityInfoDisplay.ts

```diff
@@ -187,24 +193,23 @@ const BitwiseInts = new Map([['certificatemappingmethodsraw', 2]]);
 //Here is some related documentation:
 //https://learn.microsoft.com/en-us/windows/win32/adschema/a-lastlogon
 //https://social.technet.microsoft.com/wiki/contents/articles/22461.understanding-the-ad-account-attributes-lastlogon-lastlogontimestamp-and-lastlogondate.aspx
+export const AD_NEVER_VALUE = 'NEVER';
+export const AD_UNKNOWN_VALUE = 'UNKNOWN';
```

Comment:
Easier to test against + if we change these values then we dont have to update the tests as well.